README.md

```diff
@@ -11,8 +11,14 @@ These crates are available from in (sdk):
 - azure_core
 - azure_identity
 - azure_cosmos
```

Comment:
We do need to update this to azure_data_cosmos. #482. I'll have a look today.

Reply:
I see this rename has just happened, so have updated to azure_data_cosmos